ChameleonMiniGUI/Json/MifareUltralightCardInfo.cs

```diff
@@ -30,18 +31,30 @@ public override string Uid
         public string TBO_0 { get; set; }
 
         [DataMember(Order = 3)]
-        public string Tearing { get; set; }
+        public string Pack { get; set; }
```

Comment:
Where do you see Pack in the MFU json format created in Proxmark3 client?
ref
https://github.com/RfidResearchGroup/proxmark3/blob/master/client/loclass/fileutils.c#L226

Reply:
removed